services/ui-src/src/components/menus/Sidebar.tsx

```diff
@@ -49,14 +48,9 @@ export const Sidebar = () => {
             />
           </Box>
           <Box id="sidebar-title-box" sx={sx.topBox}>
-            <Heading sx={sx.title} className={mqClasses}>
-              MCPAR Report Submission Form
-            </Heading>
+            <Heading sx={sx.title}>MCPAR Report Submission Form</Heading>
           </Box>
-          <Box
-            sx={sx.navSectionsBox}
-            className={`${mqClasses} nav-sections-box`}
-          >
+          <Box sx={sx.navSectionsBox} className={"nav-sections-box"}>
```

Comment:
Nit: className can just be a string since we're not running any JS in it. 
```suggestion
          <Box sx={sx.navSectionsBox} className="nav-sections-box">
```